### ISSUE TITLE
Mistake in experiment.py file

### ISSUE THREAD

**Issue by jarvis008** [AntixK/PyTorch-VAE#90]
In the forward function in experiment.py file, shouldn't it be: 
return self.model.forward(input, **kwargs), 
instead of 
return self.model(input, **kwargs)

If not, please explain why as I'm unable to understand. Thanks!

**Comment by folasefo** [comment]
Hi, did you encounter this problem like below? If yes, did you solve this problem? Thank you
pytorch_lightning.utilities.exceptions.MisconfigurationException: The provided lr scheduler `ExponentialLR` doesn't follow PyTorch's LRScheduler API. You should override the `LightningModule.lr_scheduler_step` hook with your own logic if you are using a custom LR scheduler.

**Comment by hanbingma** [comment]
[https://discuss.pytorch.org/t/any-different-between-model-input-and-model-forward-input/3690](url)
this link may help you.

**Comment by MisterBourbaki** [comment]
Behind the curtain, the call to model is exactly model.forward :)
